test_scripts/Polices/build_options/ATF_P_Register_NewApp_not_exist_inLocalPT_FinishPTU_PROPRIETARY.lua

```diff
@@ -80,6 +80,12 @@ function Test:Precondition_PolicyUpdateStarted_ForDefaultApplication()
   EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY" })
 end
 
+function Test:Precondition_ActivateApp()
+  local RequestId = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(RequestId)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
```

Comment:
@istoimenova, Actually activation of app is not required for PROPRIETARY flow in order to complete PTU.
Could you clarify please why it is implemented?